package.json

```diff
@@ -65,7 +66,8 @@
       "\\.(ts|tsx)$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
     },
     "moduleNameMapper": {
-      "\\.(st.css)$": "wix-ui-test-utils/stylable-proxy"
+      "\\.(st.css)$": "wix-ui-test-utils/stylable-proxy",
```

Comment:
```
"\\.(st.css|scss)$": "wix-ui-test-utils/stylable-proxy"
```
and then you can ditch the indentity-obj-proxy

Reply:
will try it out